spec/order_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'order'
+
+describe Order do
+
+  describe "#order_meal" do
+    it "lets me select multiple dishes" do
+      expect(subject.order_meal("big mac", "fries")).to eq([ "big mac", "fries" ])
```

Comment:
The thing to watch out for here is that you're testing the right / important things.  For methods like `order_meal` we probably care less about their return value than we do about what they accomplish behind the scenes.  So a better test would make sure that `order_meal` correctly updates the order, perhaps.